README.md

```diff
@@ -73,8 +75,31 @@ Change name of the asset and build path.  (gruntfile, gulp, etc)
         - ./static/node_modules/.bin/grunt ci --gruntfile static/Gruntfile.coffee
         - tar --exclude=".git" --exclude="node_modules" -czvf ~/artifact_example.tar.gz .
 
-# Running
+## Running
 
 To deploy you'll run `mina deploy`
 
 To deploy a specific branch, run `mina deploy branch=your_feature_branch`
+
+### Continuous Deployment
+
+For successful builds, CircleCI has an optional deployment step which it can run after it builds your artifacts. This example will result in CircleCI executing a deploy each time it successfully builds the `master` branch.
+
+    # circle.yml
+    deployment:
+      test:
+        branch: master
+        commands:
+          - mina deploy
+
+#### API Permissions
+
+Once this runs, mina-circle will be calling back into CircleCI's API and will require a token. Since it's running on their servers, not your own computer, you'll need to supply one to the project configuration on CircleCI. In your project's settings, find `API permissions` under `Permissions`. Enter a value for `Token label` and click `Create token`, Copy the token string and find the `Environment variables` section under `Tweaks`. That will present you with a form to add a name and value for the variable. The name of the variable should be `MINA_CIRCLE_TOKEN`, and the value is the token you copied in the previous step.
```

Comment:
_In your project's settings_ just so there's no confusion, you might try something like _In your CircleCI project settings_...